engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/ExternalTaskRestService.java

```diff
@@ -91,4 +87,10 @@
   @Produces(MediaType.APPLICATION_JSON)
   BatchDto setRetriesAsync(SetRetriesForExternalTasksDto retriesDto);
 
+  @GET
+  @Path("/topic-names")
+  @Produces(MediaType.APPLICATION_JSON)
+  List<String> getTopicNames(@QueryParam("withLockedTasks") boolean withLockedTasks, @QueryParam("withUnlockedTasks") boolean withUnlockedTasks, @QueryParam("withRetriesLeft") boolean withRetriesLeft);
```

Comment:
Lines of code must not exceed 100 characters. If lines with method signatures exceed 100 characters, each parameter must be on a new line. See [Coding Style Java General Formatting](https://github.com/camunda/camunda-bpm-dev-docs/blob/master/development/Coding-Style-Java.md#general-formatting).